pkg/storage/trb_request_attendee.go

```diff
@@ -50,23 +51,48 @@ func (s *Store) CreateTRBRequestAttendee(ctx context.Context, attendee *models.T
 
 // UpdateTRBRequestAttendee updates a TRB request attendee record in the database
 func (s *Store) UpdateTRBRequestAttendee(ctx context.Context, attendee *models.TRBRequestAttendee) (*models.TRBRequestAttendee, error) {
-	const sql = `
+	// return attendee, nil
+	stmt, err := s.db.PrepareNamed(`
 		UPDATE trb_request_attendees
-		SET
+		SET id = :id,
 			role = :role,
 			component = :component,
 			modified_by = :modified_by,
 			modified_at = CURRENT_TIMESTAMP
-		WHERE trb_request_attendees.id = :id
-	`
-	_, err := s.db.NamedExecContext(ctx, sql, attendee)
-
+		WHERE id = :id
+		RETURNING
+			id,
+			role,
```

Comment:
looks like this is missing `euaUserId` in the returning statement, so you get a blank in GQL
![image](https://user-images.githubusercontent.com/99294987/192339722-3a57f367-f741-48bd-8e6a-1f1a5c853888.png)